Makefile

```diff
@@ -138,15 +138,25 @@ endif
 endif
 
 # ---------------------------------------------------------------------
+# Electron Builder
+# ---------------------------------------------------------------------
+
+ELECTRON_BUILDER_OPTIONS=--$(TARGET_ARCH_ELECTRON_BUILDER) --extraMetadata.version=$(APPLICATION_VERSION)
```

Comment:
Yep, in fact quoting the value means that the quotes will be present on the final command. For example:

```sh
foo "--bar baz"
```